mods/cnc/rules/defaults.yaml

```diff
@@ -180,6 +180,7 @@
 			Beach: 80
 	SelectionDecorations:
 	Selectable:
+		Bounds: 12,17,0,-6
```

Comment:
Tiny nit: RA and D2k use `12,18,0,-6`.
I guess TS has bigger infantry, but these use the same size, so you can use the same bounds for consistency.